docs/source/release/v6.9.0/mantidworkbench.rst

```diff
@@ -16,33 +16,33 @@ New Features
-- Fixed bug where it was not possible to remove the grid lines on a plot once they had been added.
-- The splash screen will no longer insist on being on top of every window.
-- Fix bug that made settings window disappear behind the Workbench window on Linux when opening the font selection menu.
-- Fixed bug where a 3D plot would shrink every time you changed a property of the colorbar.
-- Fix issue when displaying large array properies in the sample logs viewer
+- Line Colour selection button is re-enabled in the toolbar of contour plots.
+- Fixed a crash which could happen if workspaces are deleted while the project saves. Save now fails in a controlled way.
+- Fixed an unhandled exception thrown when attempting to remove a curve from a plot that has a badly formatted label.
+- Fixed a bug where it was not possible to remove the grid lines on a plot once they had been added.
+- The splash screen will no longer stay on top of every window.
+- Fixed a bug that made the Settings window disappear behind the Workbench window on Linux when opening the font selection menu.
+- Fixed a bug where a 3D plot would shrink every time you changed a property of the colorbar.
+- Fixed an issue when displaying large array properties in the sample logs viewer.
 - Fixed a bug where fixed properties in the fitting view could not be un-fixed.
-- Fixed error in plot script generated for 1D MDHisto workspace plots.
-- Fix bug with ``plotSpectrum`` where quickly over-plotting could cause Workbench to hang.
+- Fixed an error in the plot script generated for 1D MDHisto workspace plots.
+- Fixed a bug with ``plotSpectrum`` where quickly over-plotting could cause Workbench to hang.
 - Mantid's package size has been reduced by removing the algorithm dialog screenshots from the documentation.
-- Fix for crash which could occur when opening file with unicode characters
-- Fix a bug where changing waterfall x and y off sets would not update the plot axes limits.
+- Fixed a crash which could occur when opening a file with unicode characters.
+- Fixed a bug where changing waterfall x and y offsets would not update the plot axes limits.
 - Fixed a bug where double clicking a plot to open the settings dialog would not end a pan/zoom event if either tool was selected.
-- Fixed crash when opening the :ref:`algm-Rebin` algorithm dialog when a group workspace is selected as the input workspace.
-- Fixed bug in the plot settings axes tab where editing an axis title and clicking ``Apply to all`` would clear your changes. The ui has been slightly reworked to make it clearer what ``Apply to all`` interacts with.
-- Fixed bug where Workbench could crash if an editor tab was closed while executing.
-- Fix for crash from setting both waterfall plot offsets to 0.
+- Fixed a crash when opening the :ref:`algm-Rebin` algorithm dialog when a group workspace is selected as the input workspace.
```

Comment:
This algorithm link might be ok as it is - but to be safe we might want to do this as it seems to be more commonly done in Mantid
```suggestion
- Fixed a crash when opening the :ref:`Rebin <algm-Rebin>` algorithm dialog when a group workspace is selected as the input workspace.
```